components/Heroes/heroes.css

```diff
@@ -0,0 +1,79 @@
+.Header {
```

Comment:
so we don't want to capitalize all css files, just css files that are directly related to a component (i.e., being imported by that component). This is because components are capitalized. This convention is due to react requiring capitalized component names, so we just capitalize all component related files to be consistent. I think this should already be the case in our files though.